.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Android Dev Container",
+  "image": "ghcr.io/eclipse-velocitas/devcontainer-base-images/base:v0.3.5",
```

Comment:
Do we have any general approach on when to use full version (like `v0.3.5`) or just short (like `v0.3`)? I notice that we in many other repos just specify `v0.3` which has the advantage that we sometimes get patches for free when rebuilding, but also that we get regressions on main due to updated components (but 3rd party license file not updated)

https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/.devcontainer/Dockerfile

Reply:
Good question indeed, this is probably something which we really should align on. Normally on Android side we were always using pinned versions - never trust anybody to really programm according to semantic versioning - could always lead to unexpected and failing builds

Reply:
> True, but that also means that you need to update every single vehicle app if we detect a vulnerability in base-image v0.3.5 and need to release v0.3.6, rather than just rebuilding all and getting the patch.

Should we really do this? I have the feeling we should release a 0.3.6 in this case anyway even if only the internally used dependencies changed. IMO we should never override a once released artifact (which changed, even if just the dependencies). 

@doosuu 
Do we already have a definition in velocitas, or should we take this into the architecture round? However, we might have to wait until Erik and me are back from vacation.

Personally I don't have a hard opinion and will go with whatever the team decides.